pkg/mdformatter/mdgen/mdgen.go

```diff
@@ -21,6 +23,8 @@ const (
 	infoStringKeyType     = "mdox-gen-type"
 	infoStringKeyExec     = "mdox-gen-exec"
 	infoStringKeyExitCode = "mdox-expect-exit-code"
+	infoStringKeyFile     = "mdox-gen-file"
```

Comment:
Hm, the only problem with metod is that what happens if one would specify both `mdox-gen-exec` and `mdox-gen-file`? (:

Reply:
I think an error needs to be shown in that case, so that one of them can be removed and there's no confusion later on what would get generated in that block. WDYT? (: